code/modules/mob/living/simple_animal/bot/floorbot.dm

```diff
@@ -389,13 +389,15 @@
 	mode = BOT_IDLE
 
 /mob/living/simple_animal/bot/floorbot/update_icon_state()
+	return
+
+/mob/living/simple_animal/bot/floorbot/update_overlays()
+	. = ..()
 	if(mode == BOT_REPAIRING)
-		icon_state = "[toolbox_color]floorbot-c"
-		return
-	if(amount > 0)
-		icon_state = "[toolbox_color]floorbot[on]"
+		. += "floorbot_work"
 	else
-		icon_state = "[toolbox_color]floorbot[on]e"
+		. += "floorbot_[on ? "on" : "off" ]"
+		. += "floorbot_[amount > 0 ? "metal" : "" ]"
```

Comment:
```suggestion
		. += "floorbot_[on ? "on" : "off"]"
		. += "floorbot_[amount > 0 ? "metal" : ""]"
```
I do think we use without spacing at the end more over the code.

Reply:
I will double check and return

Reply:
[5e2e844](https://github.com/ParadiseSS13/Paradise/pull/23081/commits/5e2e8449ed26be446b65b54245580a6eb123435b)